lib/vehicle.rb

```diff
@@ -1,18 +1,22 @@
 require 'date'
 
 class Vehicle
-  attr_reader :vin,
-              :year,
-              :make,
-              :model,
-              :engine
+  attr_reader :vin, 
+              :year, 
+              :make, 
+              :model, 
+              :engine, 
+              :registration_date
+  attr_accessor :plate_type, :registration_date
```

Comment:
It's redundant to specify both `attr_reader` and `attr_accessor` for the same attribute, `registration_date`. If you want it to be both readable and writable, you can use `attr_accessor` alone.